pkgs/development/compilers/openjdk/openjfx/deps11.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
We should keep the name of those deps.json or adjust the gitattributes file, so that github knows those are auto generated.

Reply:
done